### PR TITLE
Telcodocs 85 RN: 4.6 Release Notes description for Intel N3000 and ACC100 cards

### PR DESCRIPTION
**Pull request by kquinn1204**
Release notes description for https://issues.redhat.com/browse/CNF-1498 and https://issues.redhat.com/browse/CNF-620. 

PR preview link https://deploy-preview-33147--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-6-optimizing-data-plane-performance-with-intel-devices

### CODE REVIEW

**Comment by kquinn1204** [comment]
@SchSeba Please review the release notes description for the 4.6 release notes https://deploy-preview-33147--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-6-optimizing-data-plane-performance-with-intel-devices

**Comment by kquinn1204** [comment]
+1 this looks great!
from Sebastian

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -292,7 +292,7 @@ Note the following restrictions for {product-title} on IBM Z and LinuxONE:
 ** Automatic repair of damaged machines with machine health checking
 ** Tang mode disk encryption during {product-title} deployment
 ** {product-title} Serverless
-** Helm command-line interface (CLI) tool
+**  (CLI) tool
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:295]: Which CLI tool? I don't think that there is enough context here.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -576,6 +576,35 @@ In addition to the features provided in the previous pre-releases, this version
...
+
+[NOTE]
+====
+The N3000 Operator requires a prebuilt driver container with the Open Programmable Accelerator Engine (OPAE) out of tree kernel modules. This is built and shipped by Intel and currently supported with {product-title} 4.6.16.
+If a different {product-title} version is requested please contact Intel through their premier support portal at link:http://premiersupport.intel.com[Intel® Premier Support Access] or at mailto:openness.n3000.operator@intel.com[openness.n3000.operator@intel.com].
+
+For more details, see link:https://catalog.redhat.com/software/operators/detail/6001a748e4e3f23b0b6ad765[OpenNESS Operator for Wireless FEC Accelerators].
+====
+
+These Operators support the requirements of vRAN deployments for low power, cost and latency, whilst also delivering the capacity to manage spikes in performance for a range of use cases. One of the most compute-intensive 4G and 5G workloads is RAN layer 1 (L1) forward error correction (FEC), which resolves data transmission errors over unreliable or noisy communication channels.
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:595]: Please also apply whatever changes you make to https://github.com/openshift/openshift-docs/pull/33356 to this PR as well.

**Review by kalexand-rh — COMMENTED** [review]
I have a question, and the style items from the 4.7 PR largely apply to this one, too.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (kquinn1204) [reply release_notes/ocp-4-6-release-notes.adoc:295]: I am not sure what happened there may an erroneous key stroke but I don't believe I touched that section. I will replace with the correct 4.6 content - good spot.

Comment (kalexand-rh) [reply release_notes/ocp-4-6-release-notes.adoc:295]: Thank you! (If you have a cat, I blame it.)

**Pull request merged**